From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Black-Hole1 <bh@bugs.cc>
Date: Mon, 11 Sep 2023 15:33:17 +0800
Subject: refactor(package): disable ssh service when vm is wsl

In wsl, we can use `wsl -d machineName` join vm, no need to ssh
connection

diff --git a/package/openssh/sshd.service b/package/openssh/sshd.service
index b5e96b3a251f208f662a641544f04ee6ded492e5..c0346547a271ff6143143be9709c8a9a3b48ff6b 100644
--- a/package/openssh/sshd.service
+++ b/package/openssh/sshd.service
@@ -1,6 +1,7 @@
 [Unit]
 Description=OpenSSH server daemon
 After=syslog.target network.target auditd.service
+ConditionVirtualization=!wsl
 
 [Service]
 ExecStartPre=/usr/bin/ssh-keygen -A
